cmd/yggd/main.go

```diff
@@ -339,10 +339,13 @@ func main() {
 		if err != nil {
 			return cli.Exit(fmt.Errorf("cannot read contents of directory: %w", err), 1)
 		}
-
+		configDir := filepath.Join(yggdrasil.SysconfDir, yggdrasil.LongName)
 		env := []string{
 			"YGG_SOCKET_ADDR=unix:" + c.String("socket-addr"),
 			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+			"BASE_CONFIG_DIR=" + configDir,
```

Comment:
Given the global namespace of environment variables, I think it's better to prefix these with `YGG_`, so `YGG_BASE_CONFIG_DIR` (or even just `YGG_CONFIG_DIR`), `YGG_LOG_LEVEL` and `YGG_DEVICE_ID`. `YGG_DEVICE_ID` would be more accurate if it where named `YGG_CLIENT_ID`.

Reply:
Done.